app/services/food-service.js

```diff
@@ -63,6 +76,15 @@ class FoodService {
                     otherPlayer.id,
                     'Swap!', playerWhoConsumedFood.getHeadCoordinate(), food.color, true,
                 );
+
+                if (otherPlayerIncreaseScore) {
+                    this.playerStatBoard.setScore(otherPlayer.id, playerWhoConsumedFoodSegments.length);
```

Comment:
Innser at denne blokken er unødvendig, holder å set score. Hadde increase of decrease før, men det funket ikke så bra :P orker ikke å refaktorere nå